ddcz/views.py

```diff
@@ -1,9 +1,12 @@
 from ddcz.models.used.tavern import TavernTableVisitor
+from datetime import date
 from hashlib import md5
 import logging
+from smtplib import SMTPException
```

Comment:
Is this used anywhere?

Reply:
I guess not, not even sure what my plan was to use it. Deleted now :)